src/HealthChecks.Publisher.CloudWatch/DependencyInjection/CloudWatchHealthCheckBuilderExtensions.cs

```diff
@@ -0,0 +1,95 @@
+using Amazon;
+using HealthChecks.Publisher.CloudWatch;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    public static class CloudWatchHealthCheckBuilderExtensions
+    {
+        /// <summary>
+        /// Add a health check publisher for AWS CloudWatch.
+        /// </summary>
+        /// <remarks>
+        /// For each <see cref="HealthReport"/> published a new event <c>AspNetCoreHealthCheck</c> is sent to AWS CloudWatch with two metrics <c>AspNetCoreHealthCheckStatus</c> and <c>AspNetCoreHealthCheckDuration</c>
+        /// indicating the health check status ( 1 Healthy 0 Unhealthy)  and the total time the health check took to execute on milliseconds./>
```

Comment:
```suggestion
        /// indicating the health check status ( 1 - Healthy, 0 - Unhealthy) and the total time the health check took to execute on milliseconds.
```

Reply:
Also please remove trailing `/>` from AppInsights and DataDog publishers comments (from what you probably copy-pasted text).